src/pages/signup/StepOne.jsx

```diff
@@ -0,0 +1,78 @@
+import React from "react";
+import { TextField, Button } from "@mui/material";
+
+export default function StepOne() {
```

Comment:
Each of the steps need `props` in the parameter so we can detect click events for the next button